egui/src/containers/window.rs

```diff
@@ -91,6 +91,11 @@ impl<'open> Window<'open> {
         self.default_pos(rect.min).default_size(rect.size())
     }
 
+    pub fn fixed_pos(mut self, pos: impl Into<Pos2>) -> Self {
```

Comment:
Could do with a docstring, e.g. `/// Positions the window and prevents it from being moved`.

I should have added one to `Area::fixed_pos` but I guess I was lazy :P

Reply:
Thanks, I've added it to`Window` and `Area` both.